src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/AllServicesOrgRoleTest.java

```diff
@@ -691,4 +723,36 @@ void shouldDeletePrivateLawOrgRequestedRoleForHearing(String roleName,
             assertEquals(jurisdiction, roleAssignment.getAttributes().get("jurisdiction").asText());
         });
     }
+
+    @ParameterizedTest
+    @CsvSource({
+        "hearing-manager,SYSTEM,SSCS,PRLAPPS",
+        "hearing-viewer,SYSTEM,SSCS,PRLAPPS"
```

Comment:
For read-ability the test Jurisdiction and Case Type should probably match up: e.g. 
* PRIVATELAW => PRLAPPS
* SSCS => Benefit
i.e. the only reason in the test to expect failure should be the mis-matched process and jurisdiction: not the mis-matched case-type  (I know this is not a rule enforced here but it is in many other rule/config scenarios).

Reply:
Resolved in 14c0ad52b5d834d1ff2b0bc8584bdb5ad5cef610